src/containers/chat/active-messenger/tests/active-messenger.spec.js

```diff
@@ -79,10 +76,35 @@ describe('Active-messenger component test', () => {
         .at(1)
         .prop('message', 'Thank you for your letter, we will contact you soon')
     );
```

Comment:
What we expect in test? Fix this code smell. Add some expect method like toBe(), toEqual() ...